cmd/juju/application/deploy_test.go

```diff
@@ -127,13 +152,14 @@ var initErrorTests = []struct {
 func (s *DeploySuite) TestInitErrors(c *gc.C) {
 	for i, t := range initErrorTests {
 		c.Logf("test %d", i)
-		err := cmdtesting.InitCommand(NewDeployCommand(), t.args)
+		charmstore := &fakeCharmstoreClientShim{s.charmstore}
+		err := cmdtesting.InitCommand(NewDeployCommandForTest2(charmstore, s.charmrepo), t.args)
```

Comment:
Do we really need to change this method? Does `Init` do anything with the charmstore / charmrepo?

Reply:
Good catch. I was trying to remove all non-`*CommandForTest()` calls, but I guess that's not necessary.